src/components/GetAppPage/GetAppPage.vue

```diff
@@ -1,78 +1,84 @@
 <!-- eslint-disable vue/no-v-html -->
 <template>
   <article class="get-app-page">
-    <section class="container">
+    <section class="container top-container">
+      <img class="feature-image" src="./getapp-banner.png">
+      <img v-if="showCross" class="cross" src="./cross_3x.png">
+      
+      <div class="center app-logo">
+        <AppLogo />
+      </div>
       <h1 class="slogan center">
         <template v-if="$i18n.locale === 'zh-Hant'">
-          <LessBrowsingTC />
-          <MoreReadingTC />
+          <div class="slogan-text center"> 下載 Liker Land </div>
+          <div class="slogan-text center"> 管理你的 LikeCoin </div>
         </template>
         <template v-else>
-          <LessBrowsingEN />
-          <MoreReadingEN />
+          <div class="slogan-text center"> Download Like Land </div>
+          <div class="slogan-text center"> Manage your LikeCoin </div>
         </template>
       </h1>
-      <AppDownloadBadges
-        :from="from"
-        type="start"
-        v-bind="utmProps"
-      />
+    </section>
+    <section class="container">
+      <hr class="separate-line">
     </section>
 
     <section class="feature">
-      <div class="container side-by-side">
-        <div class="center">
-          <img class="feature-image" src="./featured-image.jpg">
+      <div>
+        <div v-for="(feature, key) in $t('GetAppPage.Feature')" :key="key" class="container side-by-side">
+          <div class="icon-part">
+            <img src="./phone_icon_3x.png" width="56px">
+          </div>
+          <div class="vp-part">
+            <div class="main-vp" v-html="feature.mainVP" /> 
+            <div class="sub-vp" v-html="feature.subVP" />
+          </div>
         </div>
-        <ul class="phone:mt-32 tablet:mt-32">
-          <li
-            v-for="(feature, key) in $t('GetAppPage.Feature')"
-            :key="key"
-            v-html="feature"
-          />
-        </ul>
       </div>
     </section>
 
-    <section class="preview container side-by-side flex phone:flex-col-reverse tablet:flex-col-reverse">
-      <div class="center phone:mt-32 tablet:mt-32">
+    <section class="container">
+      <hr class="separate-line">
+    </section>
+    <section class="footer-container">
+      <h1 class="slogan center">
+        <template v-if="$i18n.locale === 'zh-Hant'">
+          <div class="slogan-text center"> 立即下載 </div>
+        </template>
+        <template v-else>
+          <div class="slogan-text center"> Download Now </div>
```

Comment:
Why don't use localization here?

Reply:
because copied from legacy code lol 
will change it